auth-api/src/auth_api/models/affiliation.py

```diff
@@ -26,6 +26,23 @@ class Affiliation(BaseModel):  # pylint: disable=too-few-public-methods # Tempor
 
     __tablename__ = 'affiliation'
 
-    id = Column(Integer, primary_key=True)
+    id = Column(Integer, primary_key=True, autoincrement=True)
     entity = Column(ForeignKey('entity.id'), nullable=False)
     org = Column(ForeignKey('org.id'), nullable=False)
+
+    entityInfo = relationship('Entity')
+
+    @classmethod
+    def find_affiliation_by_ids(cls, org_id: int, affiliation_id: int):
```

Comment:
I don't understand how this method would be useful.  If you already have the affiliation id, you should not need to query by org id and then filter.  If it is required, it should be named differently to better distinguish it from the two methods that follow.

Reply:
Better naming is needed as Andrew suggested, but I would keep this functionality there, since affiliation is a sub-resource of org, it's always better to check the org first to avoid people fetching all affiliations without org identifier.